packages/components/src/button/stories/index.story.tsx

```diff
@@ -65,6 +65,11 @@ Default.args = {
 	children: 'Code is poetry',
 };
 
+/**
+ * Primary buttons stand out with bold color fills, making them distinct
+ * from the background. Since they naturally draw attention, each layout should contain
+ * only one primary button to guide users toward the most important action.
+ */
```

Comment:
I am reevaluating this PR from a user perspective, and I realized that we're probably not adding much value by showing a separate page with variants, including a full specimen of hover/active/disabled states. Also, some of the states are already wrong (i.e. not the same as in shipping code). For example, look at all the Link variants — they're all wrong. @auareyou Can you troubleshoot this? Are we using the Figma library components incorrectly in some way?

## What I see in the Figma embed

<img width="317" alt="Figma embed for Link variant" src="https://github.com/user-attachments/assets/84cc99ad-7397-4c1e-8786-695ee79b05ae">

## Actual Link variant (default)

<img width="122" alt="Link variant" src="https://github.com/user-attachments/assets/7884a931-39f6-4966-8072-fe3a925f40cf">

---

We already show an interactive list of all the variants in the normal story [docs](http://localhost:50240/?path=/docs/components-button--docs), so I suggest we just move the copy there. That's where all this kind of basic, important information should live. What do you think?

<img width="708" alt="Story with description" src="https://github.com/user-attachments/assets/7b2e6b28-ffc9-43f4-aea3-cc5453bd207a">

The Best Practices page is probably a good home for more supplementary information, like patterns/examples that don't quite fit in the normal story docs.

Reply:
> Are we using the Figma library components incorrectly in some way?

There's an issue in Figma where replacing underlined text layer values removes the underline 🙃 I fixed it.

> I am reevaluating this PR from a user perspective, and I realized that we're probably not adding much value by showing a separate page with variants, including a full specimen of hover/active/disabled states.

I tend to agree with this though. The value doesn't quite reflect the effort, imo.

Reply:
I moved all the variant descriptions to the main stories 👍 

Figma embeds should be a last resort, given the slow load times and maintenance cost.